.changeset/chilled-impalas-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/polaris': minor
```

Comment:
This should be a `minor` release, so these will need deleting and running through the `yarn changeset` command again. Also, we only need one changeset file per PR. Also, these go out in the official release notes, so if you can be a descriptive as possible and use sentence case, that would be fab, thanks!